File: Python/lista3 - listas/exercicio_6.py
numbers = []
for i in range(0, 4):
    numbers.append(int(input('Informe um numero inteiro: ')))

sum = 0
multiplication = 1
for i in numbers:
    sum += i
    multiplication *= i

print(numbers)
print('Soma dos numeros: %d' % sum)
print('Multiplicacao dos numeros: %d' % multiplication)

File: C/lista3 - vetores/exercicio_6.c
#include <conio.h>
#include <stdio.h>

int main() {
	int i, multiplication = 1, numbers[5], sum = 0;
	
	for (i= 0; i< 5; i++) {
		printf("Informe um numero: ");
		scanf("%d", &numbers[i]);
		
		sum += numbers[i];
		multiplication *= numbers[i];
	}
	
	printf("Total da soma dos numeros: %d \n", sum);
	printf("Total da multiplicacao dos numeros: %d \n", multiplication);
	
	getch();
	return 0;
}